src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderWidget/PaymentBuilderWidget.tsx

```diff
@@ -455,18 +453,9 @@ const PaymentBuilderWidget: FC<PaymentBuilderWidgetProps> = ({ action }) => {
           ) : (
             <>
               {finalizedAt ? (
-                <>
-                  {finalizingActions?.items[0]?.initiatorAddress ===
-                  ownerAddress ? (
-                    <FinalizeByPaymentCreatorInfo
-                      userAdddress={expenditure?.ownerAddress}
-                    />
-                  ) : (
-                    <ActionWithPermissionsInfo
-                      action={finalizingActions?.items[0]}
-                    />
-                  )}
-                </>
+                <ActionWithPermissionsInfo
```

Comment:
Hmmm? :D Alright so we always display that the payment was released via permissions, if that's what we always need to do (even if the user used their temporary payment creator permissions), we can safely remove `FinalizeByPaymentCreatorInfo`